client/src/components/lower-info-area/lower-container.vue

```diff
@@ -117,49 +117,54 @@
       </v-col>
     </v-row>
 
-    <v-row no-gutters class="px-9 contact-section">
-      <v-col cols="6" class="pr-9 pl-3 mt-3">
+    <v-row no-gutters class="contact-section">
+      <v-col cols="6">
         <v-container class="pa-9">
           <v-row>
-            <v-col cols="12" class="h3 info-underline">Need More Information?</v-col>
-            <v-col cols="12" class="copy-normal">To learn more about BC Names Request Online, please visit
-              the BC Government Homepage</v-col>
+            <v-col cols="12" class="h3 info-underline px-0">Need more information?</v-col>
+            <v-col cols="12" class="px-0">
+              <a href="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/businesses-incorporated-companies/approval-business-name" class="white--text">
```

Comment:
I recommend that you assign this to a variable in the script and just use the variable name here (so the template isn't too messy ;) ). Thanks.